vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import { URL, fileURLToPath } from 'url';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why do we need this ignore ? it should be in the devdependencies which are excluded from this rule.